.github/workflows/docker-image.yml

```diff
@@ -13,6 +13,7 @@ jobs:
         include:
           - image: alpine315
           - image: alpine316
+          - image: alpine317
```

Comment:
Do we need to support 3 versions of Alpine? Maybe remove one of them? 🤔

Reply:
let's keep the support until the distro is deprecated ?
https://endoflife.date/alpine

Reply:
Okay 👍